bootstrap/pkg/kfapp/aws/aws.go

```diff
@@ -106,6 +107,10 @@ func (aws *Aws) GetK8sConfig() (*rest.Config, *clientcmdapi.Config) {
 	return nil, nil
 }
 
+func (aws *Aws) GetKfConfig() *kfconfig.KfConfig {
```

Comment:
Why did the function signature of this method change? The original method was returning *rest.Config and clientcmdapi.Cfg?

Reply:
I don't follow, this is to provide a method to access kfconfig.
What do you mean by change?